services/src/handlers/workflows.rs

```diff
@@ -674,8 +681,8 @@ mod tests {
     use actix_web_httpauth::headers::authorization::Bearer;
     use geoengine_datatypes::collections::MultiPointCollection;
     use geoengine_datatypes::primitives::{
-        ContinuousMeasurement, FeatureData, Measurement, MultiPoint, RasterQueryRectangle,
-        SpatialPartition2D, SpatialResolution, TimeInterval,
+        ContinuousMeasurement, FeatureData, Measurement, MultiPoint, SpatialPartition2D,
```

Comment:
Please add the doc comment back to `vector_stream_websocket` (comment here, because I couldn't put it above)